xcode/X16/iOS/GameControlViewController.h

```diff
@@ -0,0 +1,16 @@
+//
+//  GameControlViewController.h
+//  CommanderX16
+//
+//  Created by Pearce, Matthew (Senior Developer) on 03/01/2020.
```

Comment:
Instead, can you write this:

    ; (C)2020 Matthew Pearce, License: 2-clause BSD

(in case you're ok with this license, otherwise reference your preferred license.)

Reply:
👍